SPECS/zsh/zsh.spec

```diff
@@ -68,42 +72,42 @@ make all html
 %check
 rm -f Test/C02cond.ztst
 make check
+
 %install
-rm -rf $RPM_BUILD_ROOT
 
 %makeinstall install.info \
-  fndir=$RPM_BUILD_ROOT%{_datadir}/%{name}/%{version}/functions \
-  sitefndir=$RPM_BUILD_ROOT%{_datadir}/%{name}/site-functions \
-  scriptdir=$RPM_BUILD_ROOT%{_datadir}/%{name}/%{version}/scripts \
-  sitescriptdir=$RPM_BUILD_ROOT%{_datadir}/%{name}/scripts \
-  runhelpdir=$RPM_BUILD_ROOT%{_datadir}/%{name}/%{version}/help
+  fndir=%{buildroot}%{_datadir}/%{name}/%{version}/functions \
+  sitefndir=%{buildroot}%{_datadir}/%{name}/site-functions \
+  scriptdir=%{buildroot}%{_datadir}/%{name}/%{version}/scripts \
+  sitescriptdir=%{buildroot}%{_datadir}/%{name}/scripts \
+  runhelpdir=%{buildroot}%{_datadir}/%{name}/%{version}/help
 
-rm -f ${RPM_BUILD_ROOT}%{_bindir}/zsh-%{version}
-rm -f $RPM_BUILD_ROOT%{_infodir}/dir
+rm -f %{buildroot}%{_bindir}/zsh-%{version}
+rm -f %{buildroot}%{_infodir}/dir
 
-mkdir -p ${RPM_BUILD_ROOT}%{_sysconfdir}
+mkdir -p %{buildroot}%{_sysconfdir}
 for i in %{SOURCE1}; do
-    install -m 644 $i $RPM_BUILD_ROOT%{_sysconfdir}/"$(basename $i .rhs)"
+    install -m 644 $i %{buildroot}%{_sysconfdir}/"$(basename $i .rhs)"
 done
 
-mkdir -p $RPM_BUILD_ROOT%{_sysconfdir}/skel
-install -m 644 %{SOURCE2} $RPM_BUILD_ROOT%{_sysconfdir}/skel/.zshrc
+mkdir -p %{buildroot}%{_sysconfdir}/skel
+install -m 644 %{SOURCE2} %{buildroot}%{_sysconfdir}/skel/.zshrc
 
 # This is just here to shut up rpmlint, and is very annoying.
```

Comment:
> This is just here to shut up rpmlint, and is very annoying. [](start = 1, length = 60)

Can you (1) verify if this is still true and (2) if true, tone the comment down to something a little more professional.

Reply:
It was no longer necessary. All of the scripts from the loop already begun with the correct `#!` and were executable.